scripts/make_release.py

```diff
@@ -212,20 +212,20 @@ def generate_changelog(component: Component, version: str):
 
     # Release notes for github
     run(
-        "rst2html.py",
+        "myst-docutils-html",
```

Comment:
Now that the release notes are in markdown, I suspect converting them to html is now unnecessary. Would you mind removing this this step and changing `draft_file` to point at `pathlib.Path(component[‘src’]) / ‘.changes.html’` instead?

Reply:
As far as I can see, `towncrier build --draft` produces Markdown now, so it's not entirely clear to me why the draft file should be `changes.html`. I'm going ahead and changing it though.

A test:

```
❯ pushd lib/esbonio && towncrier build --draft --version 0.16.3 && popd
Loading template...
Finding news fragments...
Rendering news fragments...
Draft only -- nothing has been written.
What is seen below is what would be written.

## v0.16.3 - 2023-10-29


### Breaking Changes

- Removed ``esbonio.lsp.spelling`` module, though it will be available as ``esbonio.ext.spelling`` via the ``esbonio-extensions`` package. ([#583](https://github.com/swyddfa/esbonio/issues/583))
- Drop Python 3.7 support ([#584](https://github.com/swyddfa/esbonio/issues/584))
- Drop Sphinx 4.x support ([#585](https://github.com/swyddfa/esbonio/issues/585))



  ~/Projects/Personal/esbonio  changelog-markdown !2 ·····················································································  .env 20:14:53
❯ 
```

Reply:
Thanks :) 

It doesn’t necessarily matter what the draft file is called, it just so happens that the release pipelines are [currently configured](https://github.com/swyddfa/esbonio/blob/8381ac41278b3c7b8630a059afa33f4bb6dff5e0/.github/workflows/lsp-release.yml#L64) to look for a file with that name - it seemed easier to keep the file name consistent rather than try updating all the pipelines to use something else.